src/modules/inspect/process-list/process-list.ts

```diff
@@ -5,6 +5,7 @@ import {Router} from 'aurelia-router';
 import {IIdentity} from '@essential-projects/iam_contracts';
 import {Correlation, CorrelationProcessModel, IManagementApi} from '@process-engine/management_api_contracts';
 
+import { IDiagram } from '@process-engine/solutionexplorer.contracts';
```

Comment:
Remove unneeded whitespaces
```suggestion
import {IDiagram} from '@process-engine/solutionexplorer.contracts';
```